setuptools/tests/test_virtualenv.py

```diff
@@ -69,7 +69,9 @@ def _get_pip_versions():
         'pip==9.0.3',
         'pip==10.0.1',
         'pip==18.1',
-        'pip==19.0.1',
+        # fails due to pypa/pip#6599
+        pytest.mark.xfail('pip==19.3.1', reason='fails due to pypa/pip#6599'),
```

Comment:
Sorry, I posted the previous comment from mobile and messed up the syntax. I think it should be
```suggestion
        pytest.param(
            'pip==19.3.1',
            marks=pytest.mark.xfail(reason='fails due to pypa/pip#6599'),
        ),
```

Reply:
Dang. So when I rebased and pushed for the other comment, I overwrote the patch. No matter. I'll get this change in.

Reply:
Fixed in 13ff0ba92.